src/components/vertical-menu/responsive-vertical-menu/responsive-vertical-menu-item/responsive-vertical-menu-item.component.tsx

```diff
@@ -289,6 +291,18 @@ const BaseItem = forwardRef<HTMLElement, BaseItemProps>(
       focusItem(id);
     };
 
+    const handleActionClick = useCallback(
+      (event: React.MouseEvent<HTMLAnchorElement>) => {
+        /* istanbul ignore else */
```

Comment:
suggestion (non-blocking): you could use optional chaining here to avoid the ignore but it's fine to be more explicit if you prefer 👍